tests/test_preprocessing/test_orthogonal_transformer.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import pandas as pd
+import numpy as np
+from sklego.preprocessing import OrthogonalTransformer
+
+
+@pytest.fixture
+def sample_matrix():
+    np.random.seed(1313)
+    return np.random.normal(size=(50, 10))
+
+
+@pytest.fixture
+def sample_df(sample_matrix):
+    return pd.DataFrame(sample_matrix)
+
```

Comment:
We have some default tests to see if the estimator confirms to the sklearn api

```
@pytest.mark.parametrize("test_fn", flatten([
    nonmeta_checks,
    general_checks,
    transformer_checks
]))
def test_estimator_checks(test_fn):
    test_fn(OrthogonalTransformer.__name__, OrthogonalTransformer())
```

It would be nice to add those here as well

Reply:
Added those in the latest commit